{"uuid": "5a993f6e-983a-4350-8c01-1b166c300a28", "children": ["6c542d47-957c-4439-831c-39522390a576"], "befores": [{"name": "app", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: 'geckodriver' executable needs to be in PATH. \n\n", "trace": "  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\_pytest\\fixtures.py\", line 1117, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\_pytest\\fixtures.py\", line 924, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\needl\\PycharmProjects\\python\\conftest.py\", line 30, in app\n    fixture = App(browser=browser, base_url=web_config[\"baseUrl\"])\n  File \"C:\\Users\\needl\\PycharmProjects\\python\\fixture\\app.py\", line 11, in __init__\n    self.wd = webdriver.Firefox()\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 164, in __init__\n    self.service.start()\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 83, in start\n    os.path.basename(self.path), self.start_error_message)\n"}, "start": 1597352774235, "stop": 1597352774239}], "start": 1597352774235, "stop": 1597352774311}